liberty-maven-plugin/src/it/ear-project-it/SampleEar_NonLooseApp/src/main/liberty/config/server.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<server description="jee7sample test server">
+
+    <!-- Enable features -->
+    <featureManager>
+        <feature>javaee-7.0</feature>
+    </featureManager>
+        
+    <keyStore password="keyPassword"/>
+    <basicRegistry id="basic" realm="BasicRealm">
+        <user name="admin" password="password" />
+    </basicRegistry>
+
+    <!-- To access this server from a remote client add a host attribute to the following element, e.g. host="*" -->
```

Comment:
Wondering if there should be an `<application>` element for the ear in this case since it is not loose app?

Reply:
It wasn't necessary. I was able to recreate the issue and test that the changes fixed it without having to set the `<application>` element.